src/App.js

```diff
@@ -1,21 +1,19 @@
 import React, { Component } from 'react';
+import Calendar from './components/Calendar.jsx';
+
 import './App.css';
 import { BrowserRouter as Router, Route } from "react-router-dom";
 
-// TODO: Add the components these routes map to
-// TODO: Add a navbar
+const CalendarStyle = {
```

Comment:
Make a styled-component to provide styles instead; avoid React styles when at all possible

Reply:
I have to pass the color and textColor values in the ```CalendarStyle``` object into the FullCalendar object (in ```Calendar.jsx```) as props. How can I use styled-components in this case?